Taskfile.yml

```diff
@@ -12,10 +12,15 @@ tasks:
     - cmake --build ./test/build --config Release
     - cd ./test/build && ctest -C Release --verbose
 
-  test_install:
+  install_release:
     cmds:
       - task: test_release
       - cmake --install ./test/build --config Release --prefix ./install
+      - task: clean
```

Comment:
Why cleaning after installation?

Reply:
I had test in past, that used libs or header from build directory and not from installed prefix .
This guaranties that nothing is used from build dir after install.